SourceX/miniwin/misc_io.cpp

```diff
@@ -125,29 +125,49 @@ WINBOOL SetFileAttributesA(LPCSTR lpFileName, DWORD dwFileAttributes)
 
 WINBOOL CloseHandle(HANDLE hObject)
 {
+	//Todo(Amiga): Fix the error handling

```

Comment:
This could be fixed by using -fexceptions instead of -fno-exceptions, however this will make the end result perform worse and binary get bigger on Amiga. Not that there's a lot of exception handling in DevilutionX, so we could simply remove all these ifdef's
```suggestion
```